deploy/plays/maintain.yml

```diff
@@ -0,0 +1,61 @@
+- hosts: machines
+  become: yes
+  tasks:
+    - name: Remove autolab directory completely
+      file: "dest={{ install_dir }} state=absent"
+      ignore_errors: yes
+
+    - name: Remove all images
+      docker_image:
+        name: "{{ item }}"
+        state: absent
+      ignore_errors: yes
+      with_items:
+      - "autolabjs/gitlab-ce:10.1.4-ce.0"
+      - "autolabjs/mysql:5.7.4"
+      - "autolabjs/nodejs:0.5.0"
+      - "autolabjs/executionnode:0.5.0"
+      - "main_server"
+      - "load_balancer"
+      - "execution_node"
+
+    - name: Remove firewall policies
+      import_tasks: plays/tasks/ufwreset.yml
+
+    - name: Stop containers
+      tag: stop
+      docker_container:
+        name: "{{ item }}"
+        state: stopped
+      with_items: 
+      - "autolabjs/gitlab-ce:10.1.4-ce.0"
```

Comment:
these four names are the names of the container images, not the container names. Please fetch the container names from the inventory file.

Reply:
Names of execution node containers are constructed, but the other container names are fixed. These can be read from the inventory file.